cmrtlib/linux/igfxcmrt.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+execprefix=${prefix}
+includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@/igfxcmrt
+libdir=@CMAKE_INSTALL_FULL_LIBDIR@
+
+Name: igfxcmrt
+Description: Intel(R) C for Media Runtime
+Version: @CMRT_VERSION@
+Cflags: -I${includedir} -msse4.1 -D__LINUX__ -DLINUX -std=gnu++11 -fPIC -c -rdynamic -ffloat-store
```

Comment:
I don't think that you actually can define all these cflags. I believe that what you should have should be as simple as:
`Cflags: -I${includedir}`

Here are some explanations and motivations.
1. Don't dictate to the user what they should do. That's up to the user to chose. All these:  `-msse4.1 -fPIC -c -rdynamic -ffloat-store` falls to that. That's user who selects the platform to work on. That's user who knows which target he builds and whether -fPIC is really needed, similar story with other flags. Conclusion: **drop them all!**
2. `-D__LINUX__ -DLINUX` - You do _not_ use these in the headers you have install. Thus, **simply DROP**. Look into grep:
```
[dvrogozh@dvrscl media-driver]$ fgrep -rsn LINUX /usr/include/igfxcmrt/
[dvrogozh@dvrscl media-driver]$
```
Moreover, you should NOT define these macro defs at all. If you do anywhere building cmrt  - that's a bug! What you should do is __linux__ compiler defined macro. Look here: http://www.faqs.org/docs/Linux-HOWTO/GCC-HOWTO.html#INDEX.25. That's direct guidance from Linux kernel.
3. `-std=gnu++11` - that's up to the user to select C++ standard to use! Moreover, he can select c++14 or c++17. Don't limit the user. **Drop it.**

And even more about c++11. You should not specify it in .pc file. What you can/should do is to publish some notes in README or add a requirement check in project configure which gcc or clang is required/recommended to build the project. User is obligated to follow. And we already have a note in README,md to use gcc-6 or higher. Now, look what gcc-6 is doing by default (https://gcc.gnu.org/onlinedocs/gcc-6.4.0/gcc/Standards.html#C_002b_002b-Language):
> The default, if no C++ language dialect options are given, is -std=gnu++14.

Thus. we are good without -gnu++11 for sure.

Reply:
I see. Thanks a lot for your detailed explanation. I will update it.